test/integration/client-side-encryption/driver.test.ts

```diff
@@ -297,4 +297,111 @@ describe('Client Side Encryption Functional', function () {
       });
     });
   });
+
+  describe(
+    'when @@mdb.decorateDecryptionResult is set on autoEncrypter',
+    { requires: { mongodb: '>=7.0' } },
```

Comment:
What about this test requires >= 7.0?

Reply:
https://spruce.mongodb.com/task/mongo_node_driver_next_rhel80_large_Node20_test_4.0_sharded_cluster_patch_ca2bfb0c536b665862a944c214e7cc51b49dbf1e_662fd327eb2a880007c01c39_24_04_29_17_04_51/logs?execution=0&sortBy=STATUS&sortDir=ASC

7.0 was too agressive, the $jsonSchema format I've used can't be used on 4.0 or below. I tried figuring out when this was added but it is not clear to me, do you happen to know? anyway lowered the filter requirement.